src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOperators.cs

```diff
@@ -3310,6 +3310,50 @@ .maxstack  2
             comp.VerifyIL("Program.M", il);
         }
 
+        [WorkItem(7091, "https://github.com/dotnet/roslyn/issues/7091")]
+        [Fact]
+        public void LiftedBitwiseOr()
```

Comment:
Does this pattern apply to nullable types as well?  If so I think we should add some tests for that.